cdc/cdc.cc

```diff
@@ -299,11 +307,13 @@ class transformer final {
                 {
                     auto log_ck = set_pk_columns(m.key(), batch_no, res);
                     set_bound(log_ck, rt.start);
+                    set_operation(log_ck, operation::range_delete_start, res);
```

Comment:
Should we make it range_delete_start_inclusive/exclusive to cover different bound types?

Reply:
Maybe. But let's leave this as a follow-up, no?

Reply:
Sure. Please at least add FIXME so that we don't forget.